directory_sso_api_client/models.py

```diff
@@ -8,6 +8,9 @@
 class SSOUser(AbstractUser):
     session_id = models.TextField()
     hashed_uuid = models.CharField(max_length=200)
+    has_user_profile = models.BooleanField()
+    job_title = models.CharField(max_length=123)
```

Comment:
to keep things simple we can use `models.TextField()`, which requires no `max_length`

Reply:
Done